manuscript/elsarticle-num.bst

```diff
@@ -0,0 +1,1517 @@
+%% 
+%% Copyright 2007, 2008, 2009 Elsevier Ltd
```

Comment:
This file is usually in the default latex distribution. I suppose you found this was not so on your machine?